tests/test-metadata.php

```diff
@@ -105,19 +105,44 @@ public function test_has_expanded_metadata() {
 	 * @group metadata
 	 */
 	public function test_book_information_to_schema() {
```

Comment:
I updated the old test to create an actual book with some metadata instead of passing a simple array

Reply:
I think this is a better test in a bigger context. 
One thing to consider here is we assumed that `getBookInformation` method works as expected, which is outside the scope for the `book_information_to_schema` method. In some way the `book_information_to_schema` method/unit of code that we want to test is now dependent of the `getBookInformation` method/unit. So, now we have a higher level of unit test.
Checking the code, so far we always use `getBookInformation` and `book_information_to_schema` together when the 2nd is present, so for me this tests is still good.
One thing to consider is, if we in some point use the `book_information_to_schema` without `getBookInformation` or, if we use `getBookInformation` and process in some way the data before use `book_information_to_schema`, we would like to update this probably.

Reply:
btw this is a side note, I am not suggesting any changes on this.